app/src/main/java/com/frinder/frinder/activity/LoginActivity.java

```diff
@@ -59,12 +59,15 @@ public void onCompleted(JSONObject object, GraphResponse response) {
                                     String email = object.getString("email");
                                     String id = object.getString("id");
                                     String gender = object.getString("gender");
-
+                                    String link = object.getString("link");
```

Comment:
nit: might be cleaner to move this logic to the user class (into a User.fromJSONObject method)